Ecommerce/index.css

```diff
@@ -1 +1,5 @@
-/* Write code below for ECOMMERCE TASK */
+/* Write code below for ECOMMERCE TASK */
+img {
+   justify-content: center;

```

Comment:
The 'background-color' property can be applied to the entire body or a specific container, not directly to the 'img' tag. This line should be removed.